### ISSUE TITLE
Template editor add black slash after save

### ISSUE THREAD

**Issue by Trudko** [francodacosta/atom-php-getters-setters#23]
Hi, i love the editor idea! :). Problem is I've saved this as getter template

```
%scope% function %methodName%() {
    return $this->%variable%;
}
```

but it turned into this:

```
%scope%\ function\ %methodName%()\ {
\ \ \ \ return\ $this->%variable%;
}
```

**Comment by francodacosta** [comment]
Yes, something changed on the latest versions of atom, i will try to fix 
it asap

On 04/08/2015 14:10, Trudko wrote:
>
> Hi, i love the editor idea! :). Problem is I've saved this as getter 
> template
>
> |%scope% function %methodName%() { return $this->%variable%; } |
>
> but it turned into this:
>
> |%scope%\ function\ %methodName%()\ { \ \ \ \ return\ 
> $this->%variable%; } |
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/francodacosta/atom-php-getters-setters/issues/23>.
>



---
This email has been checked for viruses by Avast antivirus software.
https://www.avast.com/antivirus

**Comment by umpirsky** [comment]
Any update on this?

**Comment by ValentinMerlet** [comment]
+1, got backslashes instead of nbsp

**Comment by tapiwan** [comment]
+1, happened to me too :)

**Comment by umpirsky** [comment]
Ping @francodacosta.

**Comment by bbailleux** [comment]
Got the problem, too.
Is there any workaround until the bug is fixed?

**Comment by umpirsky** [comment]
No afaik.

**Comment by bbailleux** [comment]
Hello @umpirsky, @Trudko, all

While trying again to get rid of those annoying back-slashes, I just discovered that I can simply remove them (actually: keep only one before every `<space>`) by editing (with caution) the `~/.atom/config.cson` file.

So I can wait for a fix without being stuck by this bug.

**Comment by Crx4** [comment]
:+1:  same here

**Comment by RenanLazarotto** [comment]
I can confirm this too.

**Comment by ldaguise** [comment]
+1
(and there is not $ before the variable name in the setter's description)

**Comment by jhernandes** [comment]
+1
